conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+class MeshlinklibConan(ConanFile):
+    name = "meshlink"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Meshlinklib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "make"
+
+    def build(self):
+        autotools = AutoToolsBuildEnvironment(self)
+        self.run("cd .. && autoreconf -fsi ")
```

Comment:
no need to add condition in conanfile.py for 3rd party library they written a generic manner so that its compatible for every platform while building for different platform

Reply:
Okay